### PR TITLE
Prepare setup for dev env WIP

### PR DESCRIPTION
**Pull request by PaulZadorozhniy**

### CODE REVIEW

File: chart/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            limits:
+              cpu: 500m
+              memory: 768Mi
+            requests:
+              cpu: 250m
+              memory: 192Mi
+          # should readinessProbe and startupProbe be present? 
+          readinessProbe:
+            httpGet:
+              path: poc-acapy-multitenant/health/ready
```
Comment (vidp1) [inline chart/templates/deployment.yaml]: Are you sure the probes are accessible on this address?

File: chart/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+              port: 8800
+            initialDelaySeconds: 15
+            periodSeconds: 10
+            failureThreshold: 25
+          startupProbe:
+            httpGet:
+              path: poc-acapy-multitenant/health/alive
```
Comment (vidp1) [inline chart/templates/deployment.yaml]: Same as above.

File: chart/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+      maxUnavailable: 0
+  selector:
+    matchLabels:
+      app: poc-acapy-multitenant
+      release: poc-acapy-multitenant
+  template:
+    metadata:
+      annotations:
+        # needs to be updated I suppose
+        iam.amazonaws.com/role: arn:aws:iam::715299534867:role/gid-dev-poc-acapy-multitenant
```
Comment (AnzeKovac) [inline chart/templates/deployment.yaml]: Do you need to access any of the AWS resources or why do you have a IAM role provided ?

File: chart/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+            value: 'https://webhook.site/86832008-51c0-4e5d-b9e0-aedcb8acf69c'
+          - name: ACAPY_MULTITENANT:
+            value: 'true'
+          - name: ACAPY_MULTITENANT_ADMIN:
+            value: 'true'
+          - name: ACAPY_MULTITENANT_JWT_SECRET:
+            value: 'sec'            
+            valueFrom:
+              secretKeyRef:
+                key: aws-chime-secret-key
```
Comment (AnzeKovac) [inline chart/templates/deployment.yaml]: This service uses chime ?

File: chart/templates/deployment.yaml (hunk 1 above)
Comment (PaulZadorozhniy) [reply chart/templates/deployment.yaml]: nope. I left a comment should it be there at all? :) if so we should do smth with it

File: chart/templates/deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+            value: 'true'
+          - name: ACAPY_TRACE_TARGET:
+            value: 'log'
+          - name: ACAPY_WALLET_STORAGE_TYPE:
+            value: 'postgres_storage'
+          - name: ACAPY_WALLET_STORAGE_CONFIG:
+            value: '{"url":"172.17.0.1:5437"}' # same as for ACAPY_ENDPOINT
+          - name: ACAPY_WALLET_STORAGE_CREDS:
+              # update with reald db values
+            value: '{\"account\":\"db_user_name\",\"password\":\"db_password\",\"admin_account\":\"admin_db_user_name\",\"admin_password\":\"admin_db_password\"}'
```
Comment (AnzeKovac) [inline chart/templates/deployment.yaml]: this should be a secret not an env

File: chart/templates/deployment.yaml (hunk 3 above)
Comment (PaulZadorozhniy) [reply chart/templates/deployment.yaml]: I don't think so, so can just remove it

File: chart/templates/service.yaml [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  name: poc-acapy-multitenant
+  labels:
+    app: poc-acapy-multitenant
+    chart: poc-acapy-multitenant-0.1.0
+    release: poc-acapy-multitenant
+    heritage: Helm
+spec:
+  type: ClusterIP
+  ports:
+    - port: 9836
```
Comment (AnzeKovac) [inline chart/templates/service.yaml:13]: on the instructions posted

PORTS="8040:8040 8041:8041"

which ones are the correct now ?

File: chart/templates/deployment.yaml (hunk 5 above)
Comment (PaulZadorozhniy) [reply chart/templates/deployment.yaml]: how to do that?

File: chart/templates/service.yaml (hunk 6 above)
Comment (PaulZadorozhniy) [reply chart/templates/service.yaml:13]: updated

Comment (PaulZadorozhniy) [reply chart/templates/service.yaml:13]: this are correct PORTS="8040:8040 8041:8041"

File: chart/templates/deployment.yaml (hunk 5 above)
Comment (AnzeKovac) [reply chart/templates/deployment.yaml:72]: You had examples in for chime

Comment (AnzeKovac) [reply chart/templates/deployment.yaml]: example
 - name: PUBNUB_SUBSCRIBE_KEY
            valueFrom:
              secretKeyRef:
                key: pubnub-subscribe-key
                name: messaging-service

**Pull request closed without merging**